.travis.yml

```diff
@@ -5,6 +5,7 @@ rvm:
 - 2.1.5
 - 2.2.0
 - 2.3.0
+- jruby-9.1.5.0
```

Comment:
JRuby seems to be going a lot of changes through releases. So, I thought it would be safer for us to control the versions on our end and test everytime we have a major JRuby release. Also, noticed that we are using multiple versions of ruby. So, eventually we might come to a point where we might need to support multiple versions of jruby. 

But, if you prefer -head I can modify that too.